lib/matplotlib/tests/test_colors.py

```diff
@@ -567,7 +567,7 @@ def test_Normalize():
     norm = plt.Normalize(1, 1 + 100 * eps)
     # This returns exactly 0.5 when longdouble is extended precision (80-bit),
     # but only a value close to it when it is quadruple precision (128-bit).
-    np.testing.assert_array_almost_equal_nulp(norm(1 + 50 * eps), 0.5)
+    assert_array_almost_equal(norm(1 + 50 * eps), 0.5, decimal=3)
```

Comment:
Part of the reason I changed to `array_almost_equal_nulp` was because of the note in the docstring: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html

However, this is more than just floating point slop comparisons, so probably OK.

Reply:
I had to pass `nulp=9_000_000_000_000` to get the two numbers to be equal locally, so I think `decimal=3` is a bit easier to understand in this case.